overview.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: index
+title: Overview
+permalink: /overview/
+---
+
+This page provides an overview of how Kroxylicious works.  For more details, please refer to the [documentation](./kroxylicious).
+
+#### What is Kroxylicious?
+
+Kroxylicious is an Apache Kafka&#174; protocol-aware proxy.  It can be used to layer uniform behaviours onto a
+Kafka based system in areas such as data-governance, security, policy enforcement and audit without needing to
+change either the applications or the Kafka Cluster.
+
+Kroxylicious is a stand-alone component which is deployed between the applications that use Kafka and
+the Kafka Cluster.  Instead of applications connecting straight to the Kafka Cluster, the applications connect to
+Kroxylious which connects to the cluster on the application's behalf. In all other respects, introducing 
+Kroxylicious into a Kafka system is transparant.
+
+To adopt Kroxylicious, there are zero code changes required to the applications. There are no additional libraries to
+install.  Kroxylicious supports applications written in any language supported by Kafka (Java, Golang, Python, Rust...).
```

Comment:
```suggestion
install.  Kroxylicious supports applications written in any language (Java, Golang, Python, Rust...).
```

Languages aren't limited to what is supported by the Kafka project (especially as it doesn't officially support Golang or Rust)

Reply:
I used the words Kafka ecosystem on the diagram, I'll use the same ways in the text but keep your parenthesised list to reinforce the point.